.vscode/.gitignore

```diff
@@ -0,0 +1 @@
+launch.json
```

Comment:
Do we want any .vscode/ files in here? I usually put a `.vscode/*` in my `.gitignores`.

Reply:
unfortunately we do need .vscode here as we need the settings.json file so the linter will work properly. It won't work properly without it because we have 2 separate projects in the same repo. 
`{
  "eslint.workingDirectories": [
      "./front-end"
  ]
}`

Reply:
Ahh yea gotcha, makes sense